ktor-client/ktor-client-curl/desktop/interop/libcurl.def

```diff
@@ -3,13 +3,20 @@ package = libcurl
 headers = curl/curl.h
 headerFilter = curl/*
 
+compilerOpts = -Idesktop/interop/include
```

Comment:
those are added in build script file: https://github.com/ktorio/ktor/blob/697582c4bf2850c75fd1b87d8a5723e215217afa/ktor-client/ktor-client-curl/build.gradle.kts#L14-L15

and yes, they could be moved inside `.def` file - there is no difference in behavior here

Reply:
Then only the linker paths are required. Will fix.